src/app/App.js

```diff
@@ -18,6 +18,12 @@ registerCoreBlocks();
 registerBlockType( UnsupportedBlock.name, UnsupportedBlock.settings );
 setUnregisteredTypeHandlerName( UnsupportedBlock.name );
 
+// disable Code and More blocks for release
+if ( ! __DEV__ ) {
```

Comment:
How does `__DEV__` get set?

Reply:
I'm surprised by the fact that the react-native documentation only has a very succinct [mention](https://facebook.github.io/react-native/docs/javascript-environment) of this global variable.
Still though, it's provided by react-native to differentiate a development build from a release one.